src/github.com/couchbase/sync_gateway/rest/server_context.go

```diff
@@ -724,12 +724,22 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		revCacheSize = db.KDefaultRevisionCacheCapacity
 	}
 
+	unsupportedOptions := &db.UnsupportedOptions{}
+	if config.Unsupported != nil {
+		if config.Unsupported.UserViews != nil {
+			if config.Unsupported.UserViews.Enabled != nil {
+				unsupportedOptions.EnableUserViews = *config.Unsupported.UserViews.Enabled
+			}
+		}
+	}
+
```

Comment:
I just realized, if

* https://github.com/couchbase/sync_gateway/pull/1499/files#diff-4cae472efa721fcbe9dd3b1e54c936a3R125 (`Unsupported        *UnsupportedConfig `)
* and https://github.com/couchbase/sync_gateway/pull/1499/files#diff-4cae472efa721fcbe9dd3b1e54c936a3R190 (`UserViews *UserViewsConfig `)

weren't pointers, then we could just do:

```
unsupportedOptions.EnableUserViews = *config.Unsupported.UserViews.Enabled
```

without any if checking.  any reason why those need to be pointers to structs?